src/utils/API.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+export default {
+  signUpUser: function ({ u }) {
```

Comment:
can we change u to user?

Reply:
changed it to the deconstructed variables being passed, so now its like
signUpUser function({uid, displayName, etc})